kubectl-plugin/pkg/cmd/get/get_cluster.go

```diff
@@ -55,7 +56,13 @@ func NewGetClusterCommand(streams genericclioptions.IOStreams) *cobra.Command {
 			if err != nil {
 				return fmt.Errorf("failed to create client: %w", err)
 			}
-			return options.Run(cmd.Context(), k8sClient)
+
+			rayClusterList, err := options.Run(cmd.Context(), k8sClient)
+			if err != nil {
+				return err
+			}
+
+			return printClusters(rayClusterList, options.ioStreams.Out)
```

Comment:
I think `printClusters` should also be called in `Run`. For unit testing purposes, we can create another function like `getRayClusters`, and `Run` can call `getRayClusters` followed by `printClusters`.

Reply:
sounds good, done